cocos/animation/animation-clip.ts

```diff
@@ -243,6 +244,18 @@ export class AnimationClip extends Asset {
         return this._additiveSettings;
     }
 
+    /**
+     * @zh 获取或设置该剪辑中存储的辅助曲线数组。
+     * @experimental
+     */
+    get auxiliaryCurveInfos_experimental () {
+        return this._auxiliaryCurveInfos as ReadonlyArray<AuxiliaryCurveInfo>;
+    }
+
+    set auxiliaryCurveInfos_experimental (value) {
```

Comment:
Setting the array directly, I am worried that there will be problems. For example, it is possible that the user will use `clip.auxiliaryCurveInfos_experimental[2] = new AuxiliarayCurveInfo();` so I suggest it is better to encapsulate the API of the operation, for example, `clip.addAuxiliarayCurve( info); clip.removeAuxiliararyCurveAt(index);` and then getter returns `readonlyArray<AuxiliararyCurveInfo>`

Reply:
I used to do that in other place (for example [animation graph layer](https://github.com/cocos/cocos-engine/blob/v3.7.3/cocos/animation/marionette/animation-graph.ts#L907-L928)). A problem is that there should be **at least 3** methods: add/remove/adjust order (even the order does not affect evaluation, just displaying). I'm also struggling with this "big 3" problem. How do you feel on this?

Reply:
I think it's OK, a little more code, but it ensures stable results and no boundary cases.

Reply:
Modified here: https://github.com/cocos/cocos-engine/pull/14659/commits/dcd64c1d0909a10c8bd4fb3e467fc68e86529261 . Please review.

Reply:
Adjusted here: https://github.com/cocos/cocos-engine/pull/14659/commits/ee8e8309e35724dcc601245f639a89110d17f16c . Please give it a review.